src/wp-admin/widgets-form.php

```diff
@@ -187,9 +187,9 @@
 
 	// Remove old position.
 	if ( ! isset( $_POST['delete_widget'] ) ) {
-		foreach ( $sidebars_widgets as $key => $sb ) {
-			if ( is_array( $sb ) ) {
-				$sidebars_widgets[ $key ] = array_diff( $sb, array( $widget_id ) );
+		foreach ( $sidebars_widgets as $key => $sidebar ) {
```

Comment:
On 82 line it use different key and value for the foreach. Can we use similar here?

```suggestion
		foreach ( $sidebars_widgets as $sidebar_id => $widgets ) {
```

Reply:
Great point! Will update this soon!

Reply:
Fixed in [5e458f2](https://github.com/WordPress/wordpress-develop/pull/3191/commits/5e458f2245b383ab3854b6d2dc0b03309a3ad0f6).